tasks/prettier_conformance/src/lib.rs

```diff
@@ -238,7 +238,8 @@ impl TestRunner {
             if snapshot_line.is_empty() { String::new() } else { title_snapshot_options }
         );
 
-        let output = Self::prettier(path, input, prettier_options);
+        let output = Self::prettier(path, input, prettier_options).replace('`', "\\`");
```

Comment:
This is confusing, I'll take a look at the root cause.

Reply:
The root cause is that the test snapshot is wrapped by `. So the template literal needs to escape
```
exports[`call.js format 1`] = ` //  you can see end of this line 
xxxxxxx
xxxxxx
xxxxxx
` // and this
```

Reply:
Does it make sense to extract these from the AST instead of doing a catch all `file.contains(output)`? Although this will require a lot of work :-(

Reply:
It should be easy to extract all the exported strings and compared them. 

We can then eventually do a `exports[file]` inside our test runner get all the strings.

Reply:
> Does it make sense to extract these from the AST instead of doing a catch all file.contains(output)? Although this will require a lot of work :-(

But that's something we need to escape anyway

Reply:
I see, let's merge. We can always improve this part later.